### PR TITLE
fix(nx-plugin): fixing linting in nx project

### PR DESCRIPTION
**Pull request by ashley-hunter**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/analogjs/analog/blob/main/CONTRIBUTING.md#-commit-message-guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Which package are you modifying?

- [ ] vite-plugin-angular
- [ ] vite-plugin-nitro
- [ ] astro-angular
- [ ] create-analog
- [ ] router
- [ ] platform
- [ ] content
- [x] nx-plugin
- [ ] trpc

## What is the current behavior?

Creating a new Nx workspace with the Analog preset adds a linting target, however it does not install the necessary eslint dependencies or create the eslintrc file at the root of the repository.

## What is the new behavior?

This utilizes the Nx eslint generator to setup linting, add dependencies and create configuration files. Also corrected the schematic description.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/nx-plugin/generators.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "app": {
       "factory": "./src/generators/app/compat",
       "schema": "./src/generators/app/schema.json",
-      "description": "Add Angular Three with proper packages and config"
+      "description": "Generates an AnalogJs application"
```
Comment (brandonroberts) [inline packages/nx-plugin/generators.json:22]:
```suggestion
      "description": "Generates an Analog application"
```

**Review by brandonroberts — APPROVED** [review]
Minor name change for Analog

File: packages/nx-plugin/generators.json (hunk 1 above)
Comment (ashley-hunter) [reply packages/nx-plugin/generators.json:22]: No problem, this was copied from the Nx generator description above, so I will change both.

**Review by brandonroberts — APPROVED** [review]

**Comment by brandonroberts** [comment]
@allcontributors add @ashley-hunter for code

**Pull request merged**